src/renderer/atlas/AtlasEngine.cpp

```diff
@@ -319,7 +324,7 @@ void AtlasEngine::WaitUntilCanRender() noexcept
 {
     if constexpr (!debugGeneralPerformance)
     {
-        WaitForSingleObjectEx(_r.frameLatencyWaitableObject.get(), 100, true);
+        WaitForSingleObjectEx(_r.frameLatencyWaitableObject.get(), INFINITE, true);

```

Comment:
this change worries me; should it?

Reply:
![image](https://user-images.githubusercontent.com/2256941/186272680-237785ce-e8d5-4904-88e4-52356bb363f8.png)

✨ No worries. ✨

Reply:
I made this change because I noticed that this `WaitForSingleObjectEx` returns within 1 V-Sync even if the window is minimized, which is weird - nothing got shown after all. In other words: OpenConsole renders even if the window is minimized, which is fairly wasteful.

...so I put `INFINITE` there, which is what the D3D demo code uses, but it didn't help either. But it surely won't hurt if we ever figure out why `frameLatencyWaitableObject` gets signaled.

If this does cause issues (e.g. because this blocks us from destroying the render thread) then we shouldn't change this `INFINITE` but rather any other potential cause, because I'm planning to add this check which might also block indefinitely: https://docs.microsoft.com/en-us/windows/win32/direct3ddxgi/waiting-when-occluded

Reply:
You know what, I'll revert this. No point in making unnecessary changes if they don't improve anything.